src/resources/postcss/views/list/_event.pcss

```diff
@@ -0,0 +1,150 @@
+.tribe-events {
+
+	/* -----------------------------------------------------------------------------
+	 * Event
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-calendar-list__event-row {
+		margin-bottom: var(--spacer-5);
+		margin-top: var(--spacer-7);
+
+		@media (--viewport-medium) {
+			margin-bottom: var(--spacer-8);
+			margin-top: var(--spacer-9);
+		}
+
+		&:last-child {
+			margin-bottom: var(--spacer-8);
+
+			@media (--viewport-medium) {
+				margin-bottom: var(--spacer-11);
+			}
+		}
+	}
+
+	.tribe-events-calendar-list__month-separator + .tribe-events-calendar-list__event-row {
+		margin-top: var(--spacer-4);
+
+		@media (--viewport-medium) {
+			margin-top: var(--spacer-7);
+		}
+	}
+
+	.tribe-events-calendar-list__event-date-tag {
+		flex: none;
+		margin-right: var(--grid-gutter-small-half-negative);
+		width: var(--grid-width-1-of-4);
+
+		@media (--viewport-medium) {
+			margin-right: 0;
+			width: var(--grid-width-1-of-9);
+		}
+	}
+
+	.tribe-events-calendar-list__event-date-tag-datetime {
+		display: flex;
+		flex-direction: column;
+		text-align: center;
+
+		@media (--viewport-full) {
+			margin-left: 28%;
```

Comment:
Is this calculated in any form? Does it look good when sliding the viewport? (I'm starting to wander about the views on sites using the page template)

Reply:
I based this roughly off the designs, and picked the `--viewport-full` breakpoint as it didn't break the text into two lines. Using page template, it may not work so well. But I think that's a question for when we do a theme pass.

Reply:
Okies. Yes, my question had more to do with using the page template and not the events template.